backend/server/index.js

```diff
@@ -1,73 +1,21 @@
 const app = require("express")();
 const server = require("http").createServer(app);
 const cors = require("cors");
-const Rooms = require("./Utils/Rooms");
+
 const io = require("socket.io")(server, { origins: "*:*" });
 const helmet = require('helmet');
 const rateLimiter = require('./rateLimiter');
+const handleSocket = require('./Utils/Socket');
 // instantiate a new rooms object to store all clients in the room
```

Comment:
Delete this comment from here and just Put this inside a socket.js file where you are creating a room object.

Reply:
Yes, there is no meaning to put room and socket.js file inside utils file,Ok I will change it